packages/cli/create-strapi-starter/utils/fetch-npm-starter.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const path = require('path');
+const execa = require('execa');
+const chalk = require('chalk');
+const stopProcess = require('./stop-process');
+
+/**
+ * Gets the package version on npm. Will fail if the package does not exist
+ * @param {string} packageName Name to look up on npm, may include a specific version
+ * @param {boolean} useYarn Yarn instead of npm
+ * @returns {Object}
+ */
+async function getPackageInfo(packageName, useYarn) {
+  // Use yarn if possible because it's faster
+  if (useYarn) {
+    const { stdout } = await execa.shell(`yarn info ${packageName} --json`);
```

Comment:
any silent option here ?

Reply:
I didn't find any in their docs. And they only allow selecting one field at a time, but it's still faster than the npm equivalent

Reply:
you are set --silent in the next yarn command though. It looks like yarn -s or yarn --silent is always valid 🤔

Reply:
I use --silent on yarn add because I found it [in their docs](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-silent) for that command

And I think --silent wouldn't be necessary for yarn here, because with the --json option it only returns the JSON object and nothing else